crates/bmp-pkt/src/wire/tests/mod.rs

```diff
@@ -1528,3 +1533,71 @@ fn test_bmp_stats() -> Result<(), BmpMessageWritingError> {
     test_write(&good, &good_wire)?;
     Ok(())
 }
+
+#[test]
+fn test_bgp_evpn() -> Result<(), BmpMessageWritingError> {
+    // TODO validate with adj in locrib adj out
+    let msg = BmpMessage::V3(BmpMessageValue::RouteMonitoring(
+        RouteMonitoringMessage::build(
```

Comment:
in correct arguments for this function: check with `cargo +nightly clippy --tests -- -Dclippy::all`